### DISCUSSION TITLE
Which installation method should I use on NixOS?

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#293] [Q&A] [answered]
Hyprland got added to Nixpkgs, so you can now install Hyprland on NixOS by simply adding it to your package list. Is there any reason to use the other installation method instead?

**Comment by fufexan** [comment, marked as answer]
Yes. The flake provides the latest changes as soon as they're pushed. Besides, we provide a NixOS module for Hyprland.
I'll also add a Home Manager module soon:tm:

Whether you use the flake or the Nixpkgs package, that's up to how fast you want to get updates for Hyprland.

**Reply by ghost** [reply]
Thanks!
